libx52/x52_control.c

```diff
@@ -22,7 +22,7 @@
 int libx52_vendor_command(libx52_device *x52, uint16_t index, uint16_t value)
 {
     int j;
-    int rc;
+    int rc = LIBUSB_ERROR_OTHER;
```

Comment:
I'd change it to 0, just to avoid the warning, but to avoid any other unforeseen side effects.